core/vm/eips.go

```diff
@@ -134,3 +135,41 @@ func enable2315(jt *JumpTable) {
 		jumps:       true,
 	}
 }
+
+// enable2929 enables "EIP-2929: Gas cost increases for state access opcodes"
+// https://eips.ethereum.org/EIPS/eip-2929
+func enable2929(jt *JumpTable) {
+	jt[SSTORE].dynamicGas = gasSStoreEIP2929
+
+	jt[SLOAD].constantGas = 0
+	jt[SLOAD].dynamicGas = gasSLoadEIP2929
+
+	jt[EXTCODECOPY].constantGas = WarmStorageReadCostEIP2929
```

Comment:
Myeah, I'd be happier to do the same for all paths that we did for SLOAD, don;t have a constant and the dynamic being cost - constant, rather just have the dynamic part return the correct value and done. I feel that's more correct because we don't charge an **additional** cold access on top of the warm, rather instead of it. Had the spec stated to always charge 100 and an additional X on cold, I'd go with this variation too, but since the spec states either one or the other, let's reflect that in the code too. Sorry, for picking on this so much.